### PR TITLE
task1: Basics. Node.js fundamental theory

### PR DESCRIPTION
**Pull request by arhigod**
imageimage
```
npm i
npm run task1_1
npm run task1_2
```
- [x] 2. Task 1.1 is fulfilled to the full extent.
- [x] 3. Task 1.2 is fulfilled to the full extent;the file is loaded fully into the RAM.
- [x] 4. Task 1.2 is fulfilled to the full extent;the file is not loaded fully in the RAM(pipeline method https://nodejs.org/api/stream.html#stream_stream_pipeline_streams_callback).
- [x] 5. All the tasks are fulfilled to the full extent (Task 1.1,Task 1.2,Task 1.3).

### CODE REVIEW

File: src/task1_1.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+process.stdin.on('data', data => {
+  console.log([...data.toString().trim()].reverse().join('') + '\n');
```
Comment (potsiahevich-dmitry) [inline src/task1_1.js]: You should put the result into write stream instead of console log

File: src/task1_2.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const readStream = createReadStream(PATH_CSV);
+const writeStream = createWriteStream(PATH_TXT);
+
+readStream.pipe(csv()).subscribe(
+  json => writeStream.write(JSON.stringify(json) + '\n'),
+  console.error,
+  () => {
+    readStream.destroy();
+    writeStream.destroy();
+  }
```
Comment (potsiahevich-dmitry) [inline src/task1_2.js]: Pay attention to the format of an example provided in the task

**Review by potsiahevich-dmitry — CHANGES_REQUESTED** [review]

File: src/task1_1.js (hunk 1 above)
Comment (arhigod) [reply src/task1_1.js]: fixed

File: src/task1_2.js (hunk 2 above)
Comment (arhigod) [reply src/task1_2.js]: fixed

**Review by potsiahevich-dmitry — APPROVED** [review]

**Pull request merged**